pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -31,9 +31,63 @@ local zotero = require('zotero')
 -- -- global state -- --
 local config = {
   client = 'zotero',
-  scannable_cite = false
+  scannable_cite = false,
+  transferable = false,
+  csl_style='',
+  dont_fill_plaincitation = false
 }
 
+-- -- -- bibliography marker generator -- -- --
+local function zotero_bibl()
+
+  if config.scannable_cite then
+    return ''
+  end
+
+  local mycsl = '{&quot;uncited&quot;:[],&quot;omitted&quot;:[],&quot;custom&quot;:[]}'
+
+  if not config.transferable then
+    return string.format(
+      '<text:section text:name=" %s">'
+        .. '<text:p text:style-name="Bibliography_20_1">'
```

Comment:
`Bibliography_20_1'  (space => %20, you see) is hard-coded in to the https://github.com/zotero/zotero-libreoffice-integration.  If a style--in this case it is an automatic style--doesn't exist  it will inherit from the 'Default Style' (usually called as 'Standard'.   I think it is safe to use what the  plugin looks for.

```
-*- mode: grep; default-directory: "~/src/zotero-libreoffice-integration/" -*-
Grep started at Mon Jul 27 13:55:42

find . -type d \( -path \*/SCCS -o -path \*/RCS -o -path \*/CVS -o -path \*/MCVS -o -path \*/.src -o -path \*/.svn -o -path \*/.git -o -path \*/.hg -o -path \*/.bzr -o -path \*/_MTN -o -path \*/_darcs -o -path \*/\{arch\} \) -prune -o \! -type d \( -name .\#\* -o -name \*.hi -o -name \*.o -o -name \*\~ -o -name \*.bin -o -name \*.lbin -o -name \*.so -o -name \*.a -o -name \*.ln -o -name \*.blg -o -name \*.bbl -o -name \*.elc -o -name \*.lof -o -name \*.glo -o -name \*.idx -o -name \*.lot -o -name \*.fmt -o -name \*.tfm -o -name \*.class -o -name \*.fas -o -name \*.lib -o -name \*.mem -o -name \*.x86f -o -name \*.sparcf -o -name \*.dfsl -o -name \*.pfsl -o -name \*.d64fsl -o -name \*.p64fsl -o -name \*.lx64fsl -o -name \*.lx32fsl -o -name \*.dx64fsl -o -name \*.dx32fsl -o -name \*.fx64fsl -o -name \*.fx32fsl -o -name \*.sx64fsl -o -name \*.sx32fsl -o -name \*.wx64fsl -o -name \*.wx32fsl -o -name \*.fasl -o -name \*.ufsl -o -name \*.fsl -o -name \*.dxl -o -name \*.lo -o -name \*.la -o -name \*.gmo -o -name \*.mo -o -name \*.toc -o -name \*.aux -o -name \*.cp -o -name \*.fn -o -name \*.ky -o -name \*.pg -o -name \*.tp -o -name \*.vr -o -name \*.cps -o -name \*.fns -o -name \*.kys -o -name \*.pgs -o -name \*.tps -o -name \*.vrs -o -name \*.pyc -o -name \*.pyo \) -prune -o  -type f \( -iname \*.java \) -exec grep --color -nH --null -e Bibliography\\\ 1 \{\} +
./build/source/org/zotero/integration/ooo/comp/ReferenceMark.java\0228:				rangeProps.setPropertyValue("ParaStyleName", "Bibliography 1");
./build/source/org/zotero/integration/ooo/comp/Document.java\0548:			// use "Bibliography 1" if available
./build/source/org/zotero/integration/ooo/comp/Document.java\0550:					paraStyleNames.getByName("Bibliography 1"));
./build/source/org/zotero/integration/ooo/comp/Document.java\0557:			styleNameContainer.insertByName("Bibliography 1", style);

Grep finished with 4 matches found at Mon Jul 27 13:55:42
```